packages/api/src/schema/api-schema.yaml

```diff
@@ -1000,11 +1000,10 @@ components:
                   properties:
                     spec:
                       oneOf:
-                        - type: "null"
+                        - type: object
```

Comment:
Why this change? I think this is changing the schema. Continuing on the other comment.

Reply:
I believe with `nullable: true` it should be fine?

Reply:
It isn't, because it will start accepting any kind of objects as well not only `null`. We would have to add `nullable` to an existing `oneOf` option here, but the problem is that the only one currently is a `$ref`.

I think there might be a way if we redefined the `spec` object here to tag it as `nullable` (maybe `$ref`'ing the `properties` field instead to avoid too much repetition)

If we decide to ignore these `null` for now, make sure to revert this to `type: "null"`